src/components/HURUmap/LocationHeader/index.js

```diff
@@ -0,0 +1,43 @@
+import { Typography, Grid, SvgIcon } from "@material-ui/core";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+const LocationHeader = ({ title, type, parent, printIcon }) => {
```

Comment:
```suggestion
const LocationHeader = ({ level, parent, title }) => {
```

1. I found out the other components use `level` instead of `type`. My bad for not checking but lets standardize on using `level` (I've update the story in PT as well).
2. We should probably accept url to the icon image rather than component (especially after what we went through with in Category PR.
3. Instead of using Material UI icons, download the icon from https://fonts.google.com/icons or even similar icon from https://wazimap-staging.netlify.app and use it until we get our own from the designer.
3. It shouldn't be an icon but a clickable button that prints the page using print CSS styles

Reply:
Very detailed feedback 👌🏾 Fixed them all @kilemensi